From 68b5029d17f157b6d6730f7f2ec55b85b6f33353 Mon Sep 17 00:00:00 2001
From: "HOME-PC\\Nik" <nikitaminyaev4@gmail.com>
Date: Sun, 18 Dec 2022 21:32:31 +0300
Subject: [PATCH] Solution

---
 Test.cpp | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 Test.cpp

diff --git a/Test.cpp b/Test.cpp
new file mode 100644
index 0000000..42e2447
--- /dev/null
+++ b/Test.cpp
@@ -0,0 +1,10 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+	cout << "Hello world!" << endl;
+
+	return 0;
+}
-- 
2.35.1.windows.2

